config/default/core.entity_form_display.node.judgement.default.yml

```diff
@@ -0,0 +1,357 @@
+uuid: d6308cb8-4066-40f5-934b-43d0c2622688
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.judgement.body
+    - field.field.node.judgement.field_case_history
```

Comment:
Should this be `field_case*s*_history`?

Reply:
Ah yes, sorry was a typo, will adjust accordingly.